spec/migrations/20170613144708_add_sti_to_container_image_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_migration
+
+describe AddStiToContainerImage do
+  let(:container_image_stub) { migration_stub(:ContainerImage) }
+
+  migration_context :up do
+    it "up" do
+      container_image = container_image_stub.create!
+      openshift_container_image = container_image_stub.create!(:size => 12345)
+
+      migrate
+
+      expect(container_image.reload).to have_attributes(:type => "ContainerImage")
+      expect(openshift_container_image.reload).to have_attributes(:type => "ManageIQ::Providers::Openshift::ContainerManager::ContainerImage")
+    end
+  end
+
+  migration_context :down do
+    it "down" do
+      container_image = container_image_stub.create!(:type => "ContainerImage")
+      openshift_container_image = container_image_stub.create!(:type => "ManageIQ::Providers::Openshift::ContainerManager::ContainerImage")
+
+      migrate
+
+      expect(container_image_stub.find(container_image.id)).not_to respond_to(:type)
+      expect(container_image_stub.find(openshift_container_image.id)).not_to respond_to(:type)
+    end
+  end
```

Comment:
No need for a down migration test, since there is no data migration.  Otherwise this test is just testing Rails itself, which isn't necessary.